.github/workflows/oracle-workflow.yml

```diff
@@ -0,0 +1,52 @@
+name: Build Oracle Connector
+
+on:
+  push:
+    paths:
+      - 'support/checkstyle/**'
+      - 'support/ide-configs/**'
+      - 'support/revapi/**'
+      - 'debezium-api/**'
+      - 'debezium-ddl-parser/**'
+      - 'debezium-assembly-descriptors/**'
+      - 'debezium-core/**'
+      - 'debezium-embedded/**'
+      - 'debezium-connector-oracle/**'
+      - 'debezium-microbenchmark/**'
+      - 'debezium-e2e-benchmark/**'
+      - 'debezium-scripting/**'
+      - 'debezium-parent/pom.xml'
+      - 'debezium-bom/pom.xml'
+      - 'pom.xml'
```

Comment:
Please add the additionl path as in https://github.com/debezium/debezium/pull/2276
Same for the PR paths